Multisig/UI/Settings/SafeSettingsViewController/SafeSettingsViewController.swift

```diff
@@ -158,6 +160,30 @@ class SafeSettingsViewController: LoadableViewController, UITableViewDelegate, U
         }
     }
 
+    func reloadSafeOwners() {
+        currentDataTask?.cancel()
```

Comment:
do you think it's good to reuse the same data task?

Reply:
I thought it would be good as these two requests are called one after another, wdyt?